README.md

```diff
@@ -8,17 +8,17 @@
 </div>
 
 <div align="center">
-  <strong>Customer-centric e-commerce on a modern stack</strong>
+  <strong>Commerce that works with your language and stack</strong>
 </div>
 
 <div align="center">
-  A headless, GraphQL commerce platform delivering ultra-fast, dynamic, personalized shopping experiences.<br/>Beautiful online stores, anywhere, on any device.
+  GraphQL native, API-only platform for scalable composable commerce.
 </div>
 
 <br>
 
 <div align="center">
-  Join our active, engaged community: <br>
+  Join our community: <br>
```

Comment:
Neither our website nor Twitter ("X"?) are ways to join the community. Maybe a better header is needed?

Reply:
Resolving for now, we can change the header in another PR.